src/openrct2/actions/FootpathSceneryRemoveAction.hpp

```diff
@@ -69,8 +69,13 @@ DEFINE_GAME_ACTION(FootpathSceneryRemoveAction, GAME_COMMAND_REMOVE_FOOTPATH_SCE
         }
 
         auto tileElement = map_get_footpath_element(_loc.x / 32, _loc.y / 32, _loc.z / 8);
-        auto pathElement = tileElement->AsPath();
+        if (tileElement == nullptr)
+        {
+            log_error("Could not find path element.");
```

Comment:
Should be `log_warning`, arguably do we need to log at all?

Reply:
Not sure about the logging part but I'll change it to log_warning, should I also change the others? Also perhaps a thought from @duncanspumpkin ?

Reply:
I prefer a log error for make results that don't have a good error message.

Reply:
While you mention that, we should probably create a few issues where we feel like they deserve better error messages.

Reply:
We are getting more and more error / warning lines showing up in the console, even just during the title screen parks. Any normal occurrence doesn't need to be logged.

Reply:
They are basically dirty todo notes. I personally like that they are coming up more. It means we aren't doing things correctly and there are areas that need to be looked at.